go/intro/if-statements/if-in-a-function.md

```diff
@@ -37,11 +37,12 @@ func main() {
 // separate function called positiveOrNegative
 // that does the same thing as the above if-else function
 func positiveOrNegative(x int) string {
-
-	if x:=1 ; x >  0 {
-		return " is positive"
+	if x > 0 {
+		return "is positive"
+	} else if x < 0 {

```

Comment:
```suggestion
	} if x < 0 {

```

Hey crosscontrol, can you remove the `else`?

The point of this insight was to teach people that they do not have to use `else` when the if is in a separate function.